modules/tile-converter/src/i3s-converter/helpers/progress.ts

```diff
@@ -0,0 +1,130 @@
+import process from 'process';
```

Comment:
Nit: You are add more hard Node.js dependencies by importing `process`. In general I prefer making code as portable as possible, even if your current usage is node only. I think some of this converter code might be useful in the browser at some point, perhaps for on-the-fly conversion. 

Also having to deal with bigints for timing makes no sense, no user is going to care about nano-second precision.

There is a portable utility is probe.gl {getHighResTimestamp} that you could use: https://github.com/uber-web/probe.gl/blob/master/modules/log/src/index.ts#L15.

You could import it or copy it.

probe.gl is designed to help measure apps, and if some API you need is missing we could add it.

Reply:
Bigint issue is fixed. Will consider using probe.gl. Thanks for the hint.